websites/rawkode.dev/src/styles/global.css

```
@@ -19,17 +17,36 @@
     font-family: var(--fonts-sans);
     font-size: 1rem;
     font-weight: 400;
     line-height: 1.6;
     color: var(--colors-text-primary);
-    background-color: var(--colors-surface-1);
+    background:
+      var(--page-glow),
+      linear-gradient(180deg, var(--colors-surface-1) 0%, color-mix(in oklch, var(--colors-surface-1) 84%, black) 100%);
+    background-attachment: fixed;
     font-feature-settings: 'kern' 1, 'liga' 1, 'calt' 1;
+    text-rendering: optimizeLegibility;
```

Stylelint (17.12.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)